core/core-workflow-steps/src/main/java/org/eclipse/sw360/antenna/workflow/processors/LicenseKnowledgeBaseResolverImpl.java

```diff
@@ -46,7 +46,7 @@ public LicenseKnowledgeBaseResolverImpl(ILicenseManagementKnowledgeBase knowledg
                 .flatMap(List::stream)
                 .map(ArtifactLicenseInformation::get)
                 .map(LicenseInformation::getLicenses)
-                .flatMap(List::stream)
+                .flatMap(Collection::stream)
```

Comment:
Just a general comment: Using bullet points in a commit message is a good hint that you're probably doing more in a single commit than you should do. If you need to enumerate things, move each point into a separate commit instead.